DOCUMENTATION.md

```diff
@@ -711,7 +717,232 @@ Or, if you want to use your own serialization method, you can use [Custom Serial
 > [API Documentation](http://hazelcast.github.io/hazelcast-nodejs-client) or GitHub repository for a required
 > interface.**
 
-## 4.1. IdentifiedDataSerializable Serialization
+## 4.1. Compact Serialization
```

Comment:
Do all of the documentation in this section apply to the Node.js client? At least "Does not require a class to implement an interface" does not apply, does it?

Reply:
Ok the sentence can be understood for the serializer instead of user class. Our serializers implement an interface. So I removed the `implementing an interface` part for Node.js